docker/conda/environments/cuda11.8_examples.yml

```diff
@@ -24,15 +24,15 @@ channels:
     - nvidia
     - conda-forge
 dependencies:
-    - boto3=1.24.59
+    - boto3=1.26.76
     - chardet=5.0.0
     - cuml=23.06
     - dask>=2023.1.1
     - dill=0.3.6
     - distributed>=2023.1.1
     - mlflow>=2.2.1,<3
     - papermill=2.3.4
-    - s3fs==2022.8.2
+    - s3fs==2023.6
```

Comment:
Lets loosen the version restriction here. Either `s3fs>=2023.6` or `s3fs=2023.6` to allow for hotfixes should work. Pinning to the exact version in conda can be problematic